terraform/cloud-platform-eks/README.md

```diff
@@ -2,6 +2,8 @@
 
 This README describes the main infrastructure components required to deliver a production-ready EKS cluster. Terraform is used as a main tool to bootstrap the infrastructure layer and EKS clusters. This terraform code requires you already have a VPC provisioned. If no VPC is provided it'll look for a VPC named as your terraform workspace 
 
+**IMPORTANT:** All cluster's names **must be unique**, each of them (EKS or kOps, doesn't matter) creates a Route53 hostzone which is unique
```

Comment:
Is this "globally unique"? If so, I'd use that phrase. If not, explain the scope within which the names must be unique.